geode-lucene/src/test/java/org/apache/geode/cache/lucene/LuceneQueriesPersistenceIntegrationTest.java

```diff
@@ -48,13 +49,13 @@
 public class LuceneQueriesPersistenceIntegrationTest extends LuceneIntegrationTest {
 
   @Rule
-  public DiskDirRule diskDirRule = new DiskDirRule();
+  public TemporaryFolder tempFolderRule = new TemporaryFolder(new File("."));
```

Comment:
Is there any reason we want to pass in a parentFolder of current dir? Can we just use the temp folder that we don't care about where it is?